### PR TITLE
Ensure tests are running with debug

### PR DESCRIPTION
**Pull request by D3nn**

### CODE REVIEW

**Review by jrryjcksn — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -139,6 +139,7 @@ integration-tests-container: cmd/wksctl/wksctl
 	NODE_OS2="ubuntu"
 	NODE_OS_CHOICE=$${NODE_OS_CHOICE:-"$$(echo $$NODE_OS1 $$NODE_OS2 | tr ' ' '\n' | shuf | head -1)"}
 	IMAGE_TAG=$(IMAGE_TAG) NODE_OS="$${NODE_OS_CHOICE}" go test -v -timeout 40m ./test/integration/container/...
```
Comment (jrryjcksn) [inline Makefile:141]: Needs to be removed!

Comment (D3nn) [reply Makefile]: Done

**Review by palemtnrider — APPROVED** [review]
LGTM

**Pull request merged**